akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -760,6 +761,29 @@ final case class `Set-Cookie`(cookie: HttpCookie) extends jm.headers.SetCookie w
   protected def companion = `Set-Cookie`
 }
 
+/**
+ * Model for the synthetic `Tls-Session-Info` header which carries the SSLSession of the connection
+ * the message carrying this header was received with.
+ *
+ * This header will only be added if it enabled in the configuration by setting
+ *
+ * ```
+ * akka.http.[client|server].parsing.tls-session-info-header = on
+ * ```
```

Comment:
Oh that's very cool then, thanks for checking (intellij didn't highlight it for me and the other one it did)